src/pages/components/menu-buttons/accessibility.mdx

```diff
@@ -40,11 +34,83 @@ accessibility considerations, some of which are described below.
 
 ### Keyboard interaction
 
+#### Menu button
+
+The menu button is set in the tab order and is activated by `Space` or `Enter`.
+When the menu is open, the first item takes focus. Focus is moved between menu
+items with the `Up` and `Down` arrow keys. `Space` or `Enter` activates the item
+with focus, which causes focus to go somewhere else and the menu to close. `Esc`
+collapses the menu and puts focus back onto the menu button.
+
+<Row>
+<Column colLg={8}>
+
+![example of menu button keyboard interaction](images/menu-button-accessibility-1.png)
+
+<Caption>
+  Menu button is reached by Tab. Space and Enter keys open the menu as well as
+  activating menu items with focus.
+</Caption>
+
+</Column>
+</Row>
+
+<Row>
+<Column colLg={8}>
+
+![illustration showing an open menu with the focus on the first item, and the arrow and Esc keys called out](images/menu-button-accessibility-2.png)
+
+<Caption>
+  When opened, the first item in the menu takes focus. Arrow keys move focus and
+  Esc closes the menu.
+</Caption>
+
+</Column>
+</Row>
+
+#### Combo button
+
+The combo button is set in the tab order where the first tab brings focus on the
+primary button and the second tab brings focus on the icon button that contains
+the menu. The respective buttons are activated by `Space` or `Enter`. When the
+menu is open, the first item takes focus. Focus is moved between menu items with
+the `Up` and `Down` arrow keys. `Space` or `Enter` activates the item with
+focus, which causes focus to go somewhere else and the menu to close. `Esc`
+collapses the menu and puts focus back onto the icon button.
+
+<Row>
+<Column colLg={8}>
+
+![example of combo button keyboard interaction](images/combo-button-accessibility-1.png)
```

Comment:
Something I just noticed for the combo button images is the label text we use for the combo button should be “Primary action” instead of “Actions”.

I see that some images on the Usage and Style tab says “Actions” for the combo button label but that is technically incorrect and should be updated at some point, because its only one primary action that can be taken.

Reply:
Thanks Lauren, I'll have those updated! ✔️